README.md

```diff
@@ -585,6 +585,8 @@ Below are listed all the variables you can customize (you may also want to take
     openwisp2_network_topology_version: "0.4"
     openwisp2_firmware_upgrader: false
     openwisp2_firmware_upgrader_version: "0.1"
+    openwisp2_monitoring: true
```

Comment:
Reminder for me: I have to update the README to add some information about the monitoring module.